src/main/kotlin/no/nav/syfo/dialogmote/api/DialogmoteEnhetApi.kt

```diff
@@ -0,0 +1,69 @@
+package no.nav.syfo.dialogmote.api
+
+import io.ktor.application.*
+import io.ktor.http.*
+import io.ktor.response.*
+import io.ktor.routing.*
+import no.nav.syfo.client.veiledertilgang.VeilederTilgangskontrollClient
+import no.nav.syfo.dialogmote.DialogmoteService
+import no.nav.syfo.dialogmote.domain.toDialogmoteDTO
+import no.nav.syfo.dialogmote.tilgang.DialogmoteTilgangService
+import no.nav.syfo.domain.EnhetNr
+import no.nav.syfo.util.*
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+private val log: Logger = LoggerFactory.getLogger("no.nav.syfo")
+
+const val dialogmoteApiEnhetUrlPath = "/enhet"
+const val dialogmoteApienhetNrParam = "enhetNr"
+
+fun Route.registerDialogmoteEnhetApi(
+    dialogmoteService: DialogmoteService,
+    dialogmoteTilgangService: DialogmoteTilgangService,
+    veilederTilgangskontrollClient: VeilederTilgangskontrollClient,
+) {
+    route(dialogmoteApiBasepath) {
+        get("$dialogmoteApiEnhetUrlPath/{$dialogmoteApienhetNrParam}") {
```

Comment:
Skal det være krøllparantes rundt `enhetNr`?

Reply:
Ja, trenger {} rund et "path variable". Kunne hatt krøllparentesen i selvet variabelen, men da jeg evt. la 2 variabler en for navnet på path-variable og en for {path-variable-navn}

Reply:
Okei, skjønner 👍